stable/kubernetes-dashboard/templates/svc.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "kubernetes-dashboard.service.fullname" . }}
```

Comment:
why allow the pods to have a name override but not the service?

Reply:
The fullname can be overridden now which applies to all resources including the service.